### PR TITLE
Bug fix in nn.feedforward

### PR DESCRIPTION
**Pull request by EnricoTrizio**
## Description
Fix bug of in_place dropout application that makes backprop fail and in batchnorm option, see #118
Also adds a backward call in test for nn.feedforward

## Status
- [x] Ready to go

### CODE REVIEW

**Comment by geraseva** [comment]
Thank you! Could you also fix batchnorm?

Replacing `self.in_features` with `(1,self.in_features)`  in https://github.com/luigibonati/mlcolvar/blob/main/mlcolvar/cvs/cv.py#L61 worked for me.

**Comment by geraseva** [comment]
Thank you!

**Review by andrrizzi — APPROVED** [review]
